cmd/ak/cmd/vars/set.go

```diff
@@ -28,7 +31,20 @@ var setCmd = common.StandardCommand(&cobra.Command{
 			return fmt.Errorf("invalid variable name %q: %w", args[0], err)
 		}
 
-		ev := sdktypes.NewVar(n, args[1], secret).WithScopeID(id)
+		var value string
```

Comment:
i think better to have a flag to read from stdin, no?

Reply:
> i think better to have a flag to read from stdin, no?

The common Unix practice is to read from stdin (`grep`, `sed` ...) if no file is specified. Or maybe use `-` as the file name. I prefer the former.

@daabr Your example is like the current behaviour. Am I missing something?

Reply:
Sorry for not being clear, my comment was just intended to provide a concrete example ("e.g.") for how we currently handle files vs stdin.

Personally, I prefer the practice of indicating the usage of stdin with  `-`, but that's not what we usually do in AK.

I'm ok with submitting your PR, but we should use a single convention in all relevant commands.

Reply:
I opened https://linear.app/autokitteh/issue/ENG-928/cli-standardization-using-stdin-instead-of-filename for follow-up

Reply:
I opened https://linear.app/autokitteh/issue/ENG-928/cli-standardization-using-stdin-instead-of-filename for follow-up